qiskit/synthesis/multi_controlled/mcx_synthesis.py

```diff
@@ -180,7 +182,7 @@ def synth_mcx_1_clean_b95(num_ctrl_qubits: int) -> QuantumCircuit:
     r"""
     Synthesize a multi-controlled X gate with :math:`k` controls using a single
     clean ancillary qubit producing a circuit with :math:`k + 2` qubits and at most
-    :math:`16 * k - 8` CX gates, by Barenco et al. [1].
+    :math:`16 * k - 24` CX gates, by Barenco et al. [1].
 
```

Comment:
this optimization method is also from [1] or should we provide another reference ?

Reply:
anyway, it will be nice to refer to a reference with the picture of the method.